src/app/directives/hover-effect.directive.ts

```diff
@@ -0,0 +1,30 @@
+import { Directive, HostListener, HostBinding, Input } from '@angular/core';
+
+@Directive({
+  selector: '[hoverEffect]'
+})
+export class HoverEffectDirective {
+
+  @Input() createdByUser!: boolean;
+
+  private _hoverBoxShadow = '1px 1px 15px 1px rgb(0 0 0 / 20%)';
+  private _hoverBackgroundColor = '#edf3fd';
```

Comment:
@KovalivNataliia Lets create a color constant, where we can add all the colors and then import constant into css and refer to all colors via variables.

```
$white: #fff;
$bright-red: #f7000f;
```
etc